app/controllers/admin/pdfs_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Admin::PdfsController < ApplicationController
+  def new
+  end
+
+  def create
+    pdf = Pdf.new(pdf_params)
+    if pdf.save
```

Comment:
I took the liberty of creating routes in the admin for pdf uploading. Feel free to change them if needed.I just needed a place to write this code.